.github/workflows/ci.yml

```diff
@@ -108,13 +107,15 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         uses: FreeRTOS/CI-CD-Github-Actions/link-verifier@main
+        with:
+          allowlist-file: ./.github/linkAllowList.txt
 
   verify-manifest:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

Comment:
Could this be simplified to match the checkout example? https://github.com/actions/checkout/tree/f43a0e5ff2bd294095638e18286ca9a3d1956744#fetch-only-the-root-files